docs/security/seccomp.md

```diff
@@ -0,0 +1,66 @@
+<!-- [metadata]>
++++
+title = "Seccomp security profiles for Docker"
+description = "Enabling seccomp in Docker"
+keywords = ["seccomp, security, docker,  documentation"]
+[menu.main]
+draft = true
```

Comment:
unrelated but this is a draft and probably shouldnt be https://raw.githubusercontent.com/docker/docker/master/docs/security/apparmor.md